### PR TITLE
Added missing Jaeger yaml file and updated skaffold.yaml to include jaeger

### PR DESCRIPTION
**Pull request by rghetia**
include jaeger.

### CODE REVIEW

File: kubernetes-manifests/jaeger.yaml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        readinessProbe:
+          periodSeconds: 5
+          tcpSocket:
+            port: 16686
+        livenessProbe:
+          periodSeconds: 5
+          tcpSocket:
+            port: 16686
+        env:
+        - name: COLLECTOR_ZIPKIN_HTT_PORT
```
Comment (sebright) [inline kubernetes-manifests/jaeger.yaml]: Should this be HTTP?

File: kubernetes-manifests/jaeger.yaml [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  name: jaeger
+spec:
+  type: ClusterIP
+  selector:
+    app: jaeger
+  ports:
+  - name: zipkin
+    port: 9411
+    targetPort: 9411
+  - name: zipking-thrift
```
Comment (sebright) [inline kubernetes-manifests/jaeger.yaml]: Should this be "zipkin-thrift"?

File: kubernetes-manifests/jaeger.yaml (hunk 1 above)
Comment (rghetia) [reply kubernetes-manifests/jaeger.yaml]: yes. fixed.

File: kubernetes-manifests/jaeger.yaml (hunk 2 above)
Comment (rghetia) [reply kubernetes-manifests/jaeger.yaml]: yes. done.

**Pull request merged**